nncf/common/quantization/initialization/range.py

```diff
@@ -204,3 +207,32 @@ def use_means_of_mins(self) -> bool:
     @property
     def use_means_of_maxs(self) -> bool:
         return not self._is_weights and not self._is_per_channel
+
+    def get_reduction_aggregation_axes(
```

Comment:
`PTRangeInitConfig` now have `get_reduction_axes`, `get_aggregation_axes` and `get_reduction_and_activation_axes`, please deal with it. I suggest to keep only `get_reduction_and_activation_axes` and overwrite it in the `PTRangeInitConfig `

Reply:
I just noticed that these functions were added in https://github.com/openvinotoolkit/nncf/pull/2117 and used only in PTQ. I have no idea why they were added inside `PTRangeInitConfig` instead of the common approach how it was done for ONNX or OpenVINO as a function of backend class. So, my understandig was that these functions are used in QAT. Now I see that they could be safely removed. 
@daniil-lyakhov is my understanding correct?

Reply:
These function are used in both QAT and PTQ quantization algorithms. These function cannot be removed as logic of QAT is differs from the logic of PTQ. Please find all references of `get_reduction_axes`, `get_aggregation_axes` to get better understanding

Reply:
Yes, you need to join `get_reduction_axes` and `get_aggregation_axes` in one `get_reduction_and_activation_axes`

Reply:
If we plan to remove `get_reduction_axes` and `get_aggregation_axes` for  `PTRangeInitCollectorParams` I don't see a reason to keep this class. What do you think of idea removing it and replacing be a common `RangeInitCollectorParams`?

Reply:
I think that reduction axes and aggregation axes are something that can not be logically separated. One has no sense without the other in current logic: there is a need to have at least one reducer and at least one aggregator.

Reply:
You've already separated these entities into a few variables. Also, not all reducers need reduction axes (for example, see NoopReducer).
Also, two atomic methods are easier to understand, develop and maintain.

Reply:
done